grid/chat/messages/profile.supr

```diff
@@ -0,0 +1,278 @@
+name = "chat/messages"
+version = "1.0.0"
+
+"""
+Get chat messages
+Retrieve paginated list of messages
+"""
+usecase GetMessages {
+	input {
+		"""
+		Destination
+		Representation of a messaging destination. Can be ID or channel name, depending on the provider capability.
+		"""
+		destination! string!
+		
+		"""
+		After Date
+		Retrieve only messages after this date and time, UNIX timestamp format (e.g. 1095379198.75) 
+		"""
+		afterDate number
```

Comment:
I think we discussed that calling it Date is misleading since it expects a timestamp, no?

```suggestion
		"""
		After Timestamp
		Retrieve only messages after the given UNIX timestamp (in seconds, e.g. 1095379198.75) 
		"""
		afterTimestamp number
```

Reply:
This is implemented in PR with discord, I'll add it here as well.